### PR TITLE
Fix 1:1 conversation title

### PR DESCRIPTION
**Pull request by mattnorris**
- Uses ID instead of email for 1:1 conversations
- Adds test
- Removes unused function 

Resolves https://jira-eng-gpk2.cisco.com/jira/browse/SSDK-1851

### CODE REVIEW

**Comment by adamweeks** [comment]
Deploy preview for *hopeful-hamilton-4090b9* ready!

Built with commit ca47b1f0bbdadbc48022ef350984cecc9b327f1b

https://deploy-preview-739--hopeful-hamilton-4090b9.netlify.com

File: packages/node_modules/@ciscospark/widget-space/src/selector.js [hunk 1]
```diff
@@ -5,23 +5,36 @@ const getWidget = (state) => state.widgetSpace;
...
+ * @param {participant} currentUser
+ * @returns {participant}
+ */
+export const getToUser = createSelector(
+  [getConversation, getCurrentUser],
+  /* eslint arrow-body-style: [0] */
+  (conversation, currentUser) => {
+    return conversation
+      .get('participants')
+      .filter((participant) => participant.get('id') !== currentUser.id)
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-space/src/selector.js]: `.find` would do the same thing as `.filter().first()`

File: packages/node_modules/@ciscospark/widget-space/src/selector.test.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import {fromJS} from 'immutable';
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-space/src/selector.test.js]: Hooray! Our first selector test!

File: packages/node_modules/@ciscospark/widget-space/src/selector.js [hunk 3]
```diff
@@ -5,23 +5,36 @@ const getWidget = (state) => state.widgetSpace;
...
+ * Get the other user in a 1:1 conversation.
+ * @param {conversation} conversation
+ * @param {participant} currentUser
+ * @returns {participant}
+ */
+export const getToUser = createSelector(
+  [getConversation, getCurrentUser],
+  /* eslint arrow-body-style: [0] */
+  (conversation, currentUser) => {
+    return conversation
```
Comment (bzang) [inline packages/node_modules/@ciscospark/widget-space/src/selector.js]: can we fix this style error and remove the eslint change?

File: packages/node_modules/@ciscospark/widget-space/src/selector.js (hunk 1 above)
Comment (mattnorris) [reply packages/node_modules/@ciscospark/widget-space/src/selector.js]: I like the efficiency. I will update.

File: packages/node_modules/@ciscospark/widget-space/src/selector.js (hunk 3 above)
Comment (mattnorris) [reply packages/node_modules/@ciscospark/widget-space/src/selector.js]: All the other functions are styled that way. I can refactor though.

Comment (mattnorris) [reply packages/node_modules/@ciscospark/widget-space/src/selector.js]: Refactored. The slight difference was the appearance of any lines of code above the `return` statement. If you have one, e.g., assignment, it's fine. If you have `return` first, it wants you to use the single line syntax.

Comment (bzang) [reply packages/node_modules/@ciscospark/widget-space/src/selector.js]: @mattnorris you should be able to do something like this:
``` js
(conversation, currentUser) => conversation
  .get('participants')
```
or 
``` js
(conversation, currentUser) =>
  conversation
    .get('participants')
```

**Pull request merged**